api/client/write.go

```diff
@@ -201,5 +231,11 @@ func (t *WriteTransaction) Delete(instanceIDs ...string) error {
 
 // end ends the active transaction.
 func (t *WriteTransaction) end() error {
-	return t.client.CloseSend()
+	if err := t.client.CloseSend(); err != nil {
+		return err
+	}
+	if _, err := t.client.Recv(); err != nil && err != io.EOF {
+		return err
```

Comment:
Buckets is now using transactions... I noticed that after ending the transaction from the client, server errors are swallowed 😨 . cc @asutula